public/js/addUsers.js

```diff
@@ -1,20 +1,25 @@
 const form = document.querySelector("#add-user-form");
 const modal = document.querySelector("#add-user-modal");
 
-form.onsubmit = async function (event) {
+form.onsubmit = function (event) {
   event.preventDefault();
-
   const formData = new FormData(form);
-  let response = await fetch("/users/create", {
-    method: "POST",
-    body: formData
-  });
 
-  let result = await response.json();
+  fetch("/users/create", {
+    method: "POST",
+    body: formData,
+  })
+    .then((response) => {
+      let result = response.json();
+      console.log(result.errors);
```

Comment:
This line shows undefined.but why?

Reply:
@tanvir077 I saw your code. There are two things here
- Need to know your motive. Are you trying to fetch errors? if it is then why you are trying to bubble up error result in then block. Those error should be in catch block.
- As per promise, when you try to reject a promise that will bubble up at catch block. If you are passing those errors as a rejected promise it should go to catch block

Reply:
my motive is can i catch the error in console log.but i cannot catch the error in the console log

Reply:
are you getting errors in catch block?

Reply:
Not sure but my understanding is you want to show errors in UI. So technically if you are getting errors in catch block then it's easy to show errors in UI with any kind of machanism. But my question is
- Are you getting errors in catch block?